src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -528,6 +528,7 @@ public function viewAction(int $orderId, Request $request): Response
             'nextOrderId' => $orderSiblingProvider->getNextOrderId($orderId),
             'paginationNum' => $paginationNum,
             'paginationNumOptions' => $paginationNumOptions,
+            'isAvailableQuantityDisplayed' => (bool) $this->configuration->get('PS_STOCK_MANAGEMENT'),
```

Comment:
```suggestion
            'isAvailableQuantityDisplayed' => $this->configuration->getBoolean('PS_STOCK_MANAGEMENT'),
```

Reply:
No because configuration is supposed to be a `ConfigurationInterface` which only contains get/set methods

`getBoolean` is a side effect of the Adapter implementation of this interface which only exists because it extends `ParameterBag` When we explicitely use the Adapter service or in service definition I don't mind using `getBoolean` but in this case it's not correct